libs/product/src/facades/product-grid/product-grid.facade.spec.ts

```diff
@@ -61,7 +63,7 @@ describe('DaffProductGridFacade', () => {
     });
 
     it('should be an observable of the list of products', () => {
-      const products = [{id: '1', name: 'Some Name'}];
```

Comment:
Originally, I was going to make the `stock_status` field required, so I fixed these tests. But then I realized that the MagentoBundleProductItemOptions (which extend DaffProduct) wouldn't have a stock status, so `stock_status` needed to be optional. And I just kept this fix instead of reverting it.